qtfred/src/CheckedDataListModel.h

```diff
@@ -141,21 +141,84 @@ class CheckedDataListModel : public QAbstractListModel
 			return defaultFlags;
 	}
 
-	inline const SCP_unordered_set<T*>& getCheckedData() {
-		return checkeds;
+private:
+	template<typename V>
+	inline void getCheckedDataInternal(SCP_unordered_set<V*> &set) {
```

Comment:
This is not a "classical" getter of a property, but instead finds all checked items and returns pointers to them. Because they can be anything from integers to mission objects, copies would be impractical here.
Perhaps renaming these three methods to `collect...` would be clearer?

Reply:
Hmmm, the rename to `collect...` may be better, yes.